index.js

```diff
@@ -18,6 +18,10 @@ class FA {
     static logEvent(name, parameters) {
         FIRAnalytics.logEvent(name, parameters);
     }
+
+    static enabled(isEnabled) {
```

Comment:
can you change this name to be either disable()/enable() or setEnabled()? so it is inline with FCM method